[2023-09-13T07:38:42.894+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-09-13T07:38:42.915+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-09-13T07:38:42.916+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:42.946+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-29 00:00:00+00:00
[2023-09-13T07:38:42.955+0000] {standard_task_runner.py:57} INFO - Started process 123 to run task
[2023-09-13T07:38:42.975+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp1o2wh31y']
[2023-09-13T07:38:42.977+0000] {standard_task_runner.py:85} INFO - Job 28: Subtask extract_data
[2023-09-13T07:38:43.128+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-29T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:43.316+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-09-13T07:38:43.709+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:43.744+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230829T000000, start_date=20230913T073842, end_date=20230913T073843
[2023-09-13T07:38:43.843+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:43.870+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:18.041+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-09-15T11:31:18.057+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-09-15T11:31:18.058+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:18.109+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-29 00:00:00+00:00
[2023-09-15T11:31:18.115+0000] {standard_task_runner.py:57} INFO - Started process 356 to run task
[2023-09-15T11:31:18.127+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpz4uajsb0']
[2023-09-15T11:31:18.130+0000] {standard_task_runner.py:85} INFO - Job 19: Subtask extract_data
[2023-09-15T11:31:18.237+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-29T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:18.429+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-09-15T11:31:18.871+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:18.904+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230829T000000, start_date=20230915T113118, end_date=20230915T113118
[2023-09-15T11:31:18.991+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:19.023+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
